app/controllers/chapters_controller.rb

```diff
@@ -219,40 +217,40 @@ def chapter_2
 
     @map_3 = {
       id: "map_3",
-      tiles_url: 'https://tiles.arcgis.com/tiles/Mj0hjvkNtV7NRhA7/arcgis/rest/services/Ch2_Fg5/VectorTileServer/tile/{z}/{y}/{x}.pbf',
+      tiles_url: 'https://tiles.arcgis.com/tiles/Mj0hjvkNtV7NRhA7/arcgis/rest/services/PP_Live_Ch2_Fg5_June19/VectorTileServer/tile/{z}/{y}/{x}.pbf',
       layers: [
         {
           id: 'over-ten-' + random_number,
           text_large: 'Over 10%',
-          source_layers: { poly: 'Ch2_Fg5_Mar' },
+          source_layers: { poly: 'ch2_eez_coverage' },
```

Comment:
Could we please extract this into a variable and reuse the same variable rather then copying over the same string?